doc/developers/contributing.rst

```diff
@@ -952,6 +952,18 @@ The easiest and recommended way to accomplish this is to
 All logic behind estimator parameters,
 like translating string arguments into functions, should be done in ``fit``.
 
+Parameters can be passed to the ``fit`` method. Generally, these parameters
+should be restricted to variables that have the shape ``n_samples``.
```

Comment:
Hi @nelson-liu ,

I've made some changes to the wording and just pushed it to this PR. Let me know what you think.